gettingstarted/Items.md

```diff
@@ -0,0 +1,26 @@
+Items
+=====
+
+Structuring Sidenote
+--------------------
+
+For any item that does something, e.g. right click functionality, stored metadata, gui, etc, you will need to create your own class that extends `Item` and use that class for everything with that item (more on that later). Some people prefer to declare everything for their item in its own class, others prefer to declare all items in one class, which is more commonly used -- slightly less code and more likely to be used in tutorials. Just note that you may see others put everything together.
+
+Making an Item
+--------------
+
+Any item, modded or vanilla, added to the minecraft world is an extension of the `Items` class from Minecraft. Start by declaring a variable of type Item, setting it as `new Item()`, thus telling minecraft that this will be an item. If you need help with (or don't know) proper naming schemes/conventions, see the Structuring documentation. The next step is to add/set an unlocalized name for your item. Unlocalized names are the strings Minecraft uses for lang files internaly. Unlocalized names are set on an item with the `.setUnlocalizedName(String)` function of the Item class; If you have a creative tab, you can put that declaration here as well (`.setCreativeTab()`).
+
+If your item is ever going to have functionality other than as a crafting component (see above note in Structuring), you will need to make your own class extending `Item` and then use that class in place of `Item()` when making the item. e.g. `testItem = new Item()` becomes `testItem = new testItemClass()`. More on extended Item classes below.
```

Comment:
This is still too java-tutorial-esque, you do not need to dumb it down this much, we can expect people to know what class extending is.